### ISSUE TITLE
[ECIP-1054] Atlantis Hardfork Outreach Tracker

### ISSUE THREAD

**Issue by soc1c** [ethereumclassic/ECIPs#130]
### TL;DR
If someone pointed you to this issue, you **probably have to upgrade your Ethereum Classic node software** in anticipation of the **upcoming Atlantis protocol upgrade** ([ECIP-1054](https://ecips.ethereumclassic.org/ECIPs/ecip-1054)).

You have to run **at least** one of the following client versions:
- Classic-Geth, version **6.0.8-stable** or later: [etclabscore/go-ethereum/releases](https://github.com/etclabscore/go-ethereum/releases)
- Multi-Geth, version **1.9.2-stable** or later: [multi-geth/multi-geth/releases](https://github.com/multi-geth/multi-geth/releases)
- Parity Ethereum, version **2.5.6-stable** or later: [paritytech/parity-ethereum/releases](https://github.com/paritytech/parity-ethereum/releases)
- _Mantis is no longer supporting Ethereum Classic!_

**Projected Hardfork Date** based on 14.5 seconds block time:
- 9:00 am UTC  |  **Friday, September 13, 2019**

Please share this on all channels. That's exactly 1 month from today creating this tracking ticket.

### Reach out to Mining Pools
- [x] https://etc.ethermine.org/
- [x] https://etc.nanopool.org/
- [x] https://ethereum-classic.miningpoolhub.com/
- [x] https://2miners.com/etc-mining-pool
- [x] https://etc.f2pool.com
- [x] https://pool.btc.com/
- [ ] https://etcpool.maxhash.org/
- [ ] https://minergate.com/etc-mining-pool
- [ ] https://etc.solopool.org/
- [ ] https://etc.arsmine.com/
- [ ] http://etc.hvpps.net/
- [ ] https://anyhash.net/
- [x] https://comining.io/
- [x] https://etc.cominers.org/
- [x] https://hashon.me/etc/home
- [ ] http://tpool.io/pool/etc/stats
- [ ] https://etc.matpool.io/
- [ ] https://beepool.org/coindetail/etc

### Update Websites and Social Media
- [x] https://ethereumclassic.org/
- [x] https://twitter.com/eth_classic?lang=en
- [ ] https://ethereumclassic.github.io/
- [ ] https://etccooperative.org/
- [x] https://etclabscore.github.io/etclabscore/
- [x] https://etclabs.org
- [x] https://twitter.com/etclabscore
- [x] https://twitter.com/etclabs
- [ ] https://en.wikipedia.org/wiki/Ethereum_Classic

### Update Block Explorers and Services
- [x] https://blockscout.com/etc/mainnet
- [x] https://gastracker.io/
- [x] https://etcblockexplorer.com/home
- [x] https://etherhub.io/home
- [ ] https://0xeth.com/
- [x] https://www.ethercluster.com/

### Reach out to Exchanges
- [ ] https://hk.coineal.com/trade_center.html?s=etc-btc#en_US
- [ ] https://www.cointiger.com/en-us/#/trade_pro?coin=etc_btc
- [x] https://trans.zb.com/etcusdt
- [x] https://trade.coinegg.com/trade/etc_btc
- [ ] https://www.bitker.com/#/bbTrades/etc_btc
- [ ] https://www.catex.io/trading/ETC/TRX
- [ ] https://www.idcm.io/trading/ETC_USDT
- [x] https://www.okex.com/market?product=etc_usdt
- [ ] https://www.digifinex.com/en-ww/trade/USDT/ETC
- [ ] https://www.55.com/home/#/exchange?symbol=ETCETH&site=B
- [ ] https://www.coinbene.com/exchange/en_US/ETCUSDT
- [ ] https://www.bitforex.com/en/spot/etc_usdt
- [ ] https://www.bibox.com/exchange?coinPair=ETC_USDT
- [ ] https://hitbtc.com/ETC-to-BTC
- [ ] https://www.digifinex.com/en-ww/trade/ETH/ETC
- [ ] https://www.idax.pro/exchange?pairname=ETC_USDT
- [ ] https://p2pb2b.io/trade/ETC_USD
- [ ] https://www.hbg.com/etc_usdt/exchange/
- [ ] https://coinsbit.io/trade/ETC_BTC
- [x] https://www.binance.com/en/trade/ETC_USDT
- [x] https://pro.coinbase.com/trade/ETC-USD
- [x] https://www.bitfinex.com/
- [x] https://international.bittrex.com/Market/Index?MarketName=BTC-ETC
- [x] https://poloniex.com/exchange#btc_etc
- [x] https://trade.kraken.com/markets/kraken/etc/eur

**Comment by saturn-network** [comment]
Saturn Wallet and our dApps are ready for #atlantis!

**Comment by igorbarinov** [comment]
Cool. BlockScout archive node is updated to 2.6.1.

**Comment by TokenHash** [comment]
As of Aug 22, 2019 these are the stakeholders who have confirmed that I am aware of:

• Cointron
• Circle/Poloniex
• CoinEgg
• Coinbase
• Bitfinex
• 2miners
• NanoPool
• BlcokScout
• ShapeShift
• Saturn/P3C/DappDirect
• Ethernode

Large stakeholders who did not confirm yet:

• Ethermine
• MiningPoolHub
• Kraken
• Binance

**Comment by Max-Casanova** [comment]
@soc1c 
https://baikalmine.com
Are ready to a fork!

**Comment by adaminers** [comment]
https://pps-etc.adaminers.com
https://solo-etc.adaminers.com 

We are ready for the fork!

**Comment by xiphon** [comment]
https://etc.fairpool.xyz is ready

**Comment by bcmob** [comment]
POOL [https://simplepool.link/etc](https://simplepool.link/etc) is ready
parity 2.5.7-stable and Classic-Geth  6.0.9

**Comment by mathias7799** [comment]
https://etc.protonmine.com is ready for the fork :)

**Comment by Athethepainter** [comment]
Hi everyone, 

[https://www.cruxpool.com/etc](url) is ready for the fork !

**Comment by malefol7** [comment]
Hi everyone,
cominers.org is ready, update performed successfully, parity 2.5.7 stable

**Comment by HashonPool** [comment]
@soc1c
HashOn-Pool
https://hashon.me
Are prepared for Atlantis fork!

**Comment by soc1c** [comment]
Done

**Comment by zmitton** [comment]
This page is a great resource. What can we do to save it? we are going to need it for every hard fork

I suggest committing it to the repo somehow so we all have it.
